uktrade/data-hub-frontend
# Reactify investment projects collection list

## Description of change
This replaces our current implementation of collection list pages. This removes our use of nunjucks and introduces React/Redux to give us a reusable component that can be applied to all other collection list pages.

## Test instructions
1. Go to `/investments/projects` - You should be able to filter with a typeahead, page, sort and download results as per current collection page.
2. Go to `/investments/profiles` the <CollectionList> component is reused but only shows results with a header as per live version.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
